src/main/webapp/repo-bootstrap/global/configuration/organization-permission-mappings-config.xml

```diff
@@ -0,0 +1,52 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+    This file contains global permissions configuration for Crafter Studio. Permissions per site are managed
+    within Crafter Studio's UI.
+
+    The structure of this file is:
+    <permissions>
+        <site id="###GLOBAL###"> (global management)
+            <role name="">
+                <rule regex="/.*">
+                    <allowed-permissions>
+                        <permission>Read</permission>
+                        <permission>Write</permission>
+                        <permission>Delete</permission>
+                        <permission>Create Folder</permission>
+                         <permission>Publish</permission>
+                    </allowed-permissions>
+                </rule>
+            </role>
+        </site>
+    </permissions>
+
+    This binds a set of permissions to a role globally for the entire application.
+-->
+
+<permissions>
+    <role name="crafter-admin">
+        <rule regex="/.*">
+            <allowed-permissions>
+                <permission>Read</permission>
+                <permission>Write</permission>
+                <permission>Delete</permission>
+                <permission>Create Folder</permission>
+                <permission>Publish</permission>
+            </allowed-permissions>
+        </rule>
+    </role>
+    <role name="create-sites">
```

Comment:
These are old permissions/roles that don't exist anymore. What is the point of this file?

Reply:
I added that one long time ago, needs to be removed

Reply:
K, will wait for the PR